chromadb/telemetry/posthog.py

```diff
@@ -21,10 +22,34 @@ def __init__(self, system: System):
         posthog_logger = logging.getLogger("posthog")
         # Silence posthog's logging
         posthog_logger.disabled = True
+
+        self.batched_event: Optional[TelemetryEvent] = None
+        self.batch_size = 0
+
         super().__init__(system)
 
     @override
     def capture(self, event: TelemetryEvent) -> None:
```

Comment:
addressed elsewhere, but i think firing the first event on a collection with batch-size one is a good compromise

Reply:
Done

Reply:
In the case of add, I think we would want to fire on the first call to add() per collection not per client, which is the case as written now.

Reply:
Done. I gave `TelemetryEvent` a `batch_key` property -- for `CollectionAdd` it contains the UUID and the name `CollectionAdd`. If posthog hasn't seen an event's `batch_key` it sends the event immediately. This also let me get rid of `can_batch()` since two events can be batched iff they have the same `batch_key`